### ISSUE TITLE
keyboard dialog: convurt the (Speak typed characters) and the (speeck typed words) option to a single combo box instead of 2 check boxes, and 1 shortcut instead of 2 shortcuts.

### ISSUE THREAD

**Issue by amirmahdifard** [nvaccess/nvda#14653]
### Is your feature request related to a problem? Please describe.
currently, we have these 2 options as 2 check boxes, and there are 2 shortcuts are a sighnd to them, nvda + 2, and nvda + 3 to toggel them: but, please convurt them to a combo box, and if you do this, you can get one of the shortcuts used for something else: if you turn these options in to a single combo box, then you can have only nvda + 2 for it, or nvda + 3: and when the user presses this shortcut, instead of saying "on" or "off", it would be like this as all other combo boxes: speeck typed characters: speeck typed words and characters: speeck typed words, and off: so, as you can see, this will become as simple as this, and the other shortcut could be used for something else later even.
### Describe the solution you'd like
as discribed abuv, i'd like to have these 2 options in a single combo boxes because it's something that we can configger it in more than one way, and also more simple and less confusing with only 1 shortcut for it.
### Describe alternatives you've considered
for now we have check boxes for these options, and 2 difrent shortcuts for them, wich works generally but it would be more nice if this one happened as i explained why it would be bettar.
### Additional context
as i know, this would breake compatability, so i think that it should be done in something like 2024.1, unless the nvaccess team has a way to develop them quicker than that: as by now, any of my ideas haven't been developed in nvda yet, so i'm excitedly waiting for it to happen: thanks.

**Comment by gregjozk** [comment]
agree, there have already been implemented other changes which merged functions, which were splitted before like reporting of indentations etc.

**Comment by Adriani90** [comment]
This is not desirable now that we have speak typed cahracters and words for different use cases, namely everywhere, in edit fields only, and off.
I am closing as won't fix since both speak typed characters and words are now two separate combo boxes.
